packages/ui/src/components/PropertiesModal/PropertiesModal.test.tsx

```diff
@@ -0,0 +1,226 @@
+import { screen } from '@testing-library/dom';
+import { render } from '@testing-library/react';
+
+import { ITile } from '../Catalog';
+import { PropertiesModal } from './PropertiesModal';
+
+describe('Component tile', () => {
+  const tile: ITile = {
+    type: 'component',
+    name: 'atom',
+    title: 'Atom',
+    description: 'Poll Atom RSS feeds.',
+    headerTags: ['Stable'],
+    tags: ['document', '4.0.0'],
+    rawObject: {
+      component: {
+        kind: 'component',
+        name: 'atom',
+        title: 'Atom',
+        description: 'Poll Atom RSS feeds.',
+      },
+      properties: {
+        bridgeErrorHandler: {
+          kind: 'property',
+          displayName: 'Bridge Error Handler',
+          required: false,
+          type: 'boolean',
+          defaultValue: false,
+          description: 'Allows for bridging the consumer to the Camel',
+        },
+      },
+    },
+  };
+
+  it('renders property modal correctly', () => {
+    // modal uses React portals so baseElemnt needs to be use here
+    const { baseElement } = render(<PropertiesModal tile={tile} isModalOpen={true} onClose={jest.fn()} />);
+    expect(baseElement).toMatchSnapshot();
+
+    // info
+    expect(baseElement.getElementsByClassName('pf-v5-c-modal-box__title-text').item(0)).toBeInTheDocument();
```

Comment:
Can we use data-testid attributes instead? The thing is this prefix will change with the next version of patternfly

Reply:
I am not sure if I can set data-testid directly in that element since that element is created by Patternfly if I understand that correctly and the title text is set via `title` property in `Modal` component ( https://github.com/mkralik3/kaoto-next/blob/catalog_modal/packages/ui/src/components/PropertiesModal/PropertiesModal.tsx#L36 ). But I noticed in the Doc that also `titleLabel` can be set, so, I can add there a custom label.

Reply:
Or I can change it to `expect(baseElement.getElementsByTagName('header').item(0))` [but still, if PF changes the implementation, it don't need to be in the header element]

Reply:
I think that just checking for the test would be enough as the in-depth tests around the `Modal` component already happen on `@patternfly`'s side :smiley: